### ISSUE TITLE
Update to the latest version of webcomponents.js

### ISSUE THREAD

**Issue by jpnelson** [skatejs/web-components#5]
Unfortunately there are breaking changes. I tried upgrading to the latest sha, which was https://github.com/webcomponents/webcomponentsjs/commit/393764a520cd213b06dae3285444c61ef7fae7ac , but it failed.

My attempt to upgrade: https://github.com/skatejs/web-components/pull/3
The failed upgrade in skatejs: https://github.com/skatejs/skatejs/pull/748
My revert of it: https://github.com/skatejs/web-components/pull/4

For now I'm going to have to fork webcomponents.js and skatejs-web-components so that I can get the fix that I wanted out of this. We should be able to upgrade eventually, but the failure is due to a breaking change (disconnected / attached were renamed to connected / disconnected in v1 webcomponents, so upgrading would mean skate 2.0, which I imagine would be unwise while we're waiting on the v1 spec).

@treshugart , I'm just going to depend on my forks for now, but if you have an opinion of what we should do, I'd love to know.

**Comment by treshugart** [comment]
As a side note, in a separate pr I'm trying
webreflection/document-register-element which is semantically versions and
supports v0 and v1.

**Comment by thepian** [comment]
I'm confused. Is web components just not concerned with providing an interface? I'm just dipping my toe in, but so far my conclusions are:

1) x-tag, doesn't work since it cannot put the methods on the custom element
2) webcomponents.js doesn't provide the window.customElements interface or document which one to use
3) skate.js uses odd dependencies to web components fork

And we are supposed to make real software based on this. It really looks to me like the best strategy for developing anything is to leave this for another year or two until more browsers support it. If there is no stable API that will use the native support where possible without introducing bloat(yes I'm looking at you Polymer), then it's just a proof of concept.

**Comment by treshugart** [comment]
> skate.js uses odd dependencies to web components fork

There are active PRs into their V1 branch that will get pulled in. As I stated in your Skate issue, I'm working on pulling in webreflection/document-register-element but there is a blocking issue with old IE.

Skate itself is 100% stable (and uses native where available). Its only the polyfill situation that is temporarily in flux. I anticipate it being resolved in the next week or two. Until then I'd recommend pulling in the polyfills yourself for the time being. Sorry for the inconvenience.

**Comment by treshugart** [comment]
Closing as we've now released the one with the versioned polyfill. @thepian we'll be releasing a Skate version soon that supports this new version and also IE 9 and 10.
